src/common/display.c

```diff
@@ -99,6 +99,19 @@ static void guac_common_display_free_layers(guac_common_display_layer* layers,
 
 }
 
+/**
+ * Allocates a display and a cursor which are used to represent the remote
+ * display and cursor. If the allocation fails then the function returns NULL.
```

Comment:
Same here - much better, but return values should also be documented with `@return`.

Reply:
Same here - done